app/controllers/ctc/questions/life_situations2020_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Ctc
+  module Questions
+    class LifeSituations2020Controller < QuestionsController
+      # TODO: Transition to Authenticated once we log in client
+      include AnonymousIntakeConcern
+
+      layout "intake"
+
+      private
+
+      def illustration_path; end
+
+      def next_path
+        @form.cannot_claim_me_as_a_dependent != "yes" ? questions_use_gyr_path : super
```

Comment:
This is how I achieve:
> - if "No one can claim me as a dependent" is NOT checked, route to /use-GYR
> - If "No one can claim me as a dependent" IS checked, route to /filing-status

Reply:
i would strongly recommend against using 2020 in the URL because then we have to go back and update urls when we start filing for 2021 instead of just being able to update the contents of the page. Some of the pages we added, the whole thing is super core to being from that year so its a grey area but this one doesnt feel that way to me -- /questions/life-situations should be sufficient.

Reply:
ah, i guess there is the life situations 2019 controller. thats a real bummer.

Reply:
Oh! one thing. you can write a little helper method on the form if you want to if you think the != yes makes things kind of hard to read 

```
def cannot_be_claimed_as_dependent?
  cannot_be_claimed_as_dependent == "no"
end
```